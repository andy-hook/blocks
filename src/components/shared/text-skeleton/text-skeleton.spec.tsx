import React from "react"
import TextSkeleton from "./text-skeleton"
import { renderWithTheme } from "@test-utils"

describe("<LabelText />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <TextSkeleton />)
    expect(tree).toMatchSnapshot()
  })
})
