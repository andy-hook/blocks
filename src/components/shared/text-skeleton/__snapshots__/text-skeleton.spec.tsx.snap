// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LabelText /> renders correctly 1`] = `
<div
  className="sc-bdVaJa hqpQZd"
  width="md"
>
  Â 
</div>
`;
